packages/docusaurus-theme-classic/src/theme/Layout/index.tsx

```diff
@@ -43,6 +45,18 @@ function Layout(props: Props): JSX.Element {
       </div>
 
       {!noFooter && <Footer />}
+
+      <Head
```

Comment:
I don't feel like it is a good solution.

If we put this here, this means users won't be able to override global site config.metadatas by using sizzling

To me, the problem is related to having `{pageImage && <meta name="twitter:card" content="summary_large_image" />}` in the Seo component.

DocItem uses Seo in a nested way, so when declaring its image, it overrides the site global `twitter:card` meta by mistake.

I think hardcoding the `twitter:card` in LayoutHead is a safer solution that should also work, and would allow users to override it globally with site config and also on a page-by-page basis with sizzling

Reply:
I see, you are right, fixed this.